### DISCUSSION TITLE
E21: Cannot make changes, 'modifiable' is off

### DISCUSSION THREAD

**Discussion by sshiradwade-synamedia** [LazyVim/LazyVim#417] [Q&A]
I'm getting `E21: Cannot make changes, 'modifiable' is off` when I try to install the plugin with "I"

<img width="1392" alt="Screenshot 2023-03-12 at 3 07 13 PM" src="https://user-images.githubusercontent.com/60691696/224546776-e284ab16-6672-4b0e-8008-d58256708796.png">

**Comment by folke** [comment]
probably because you added options in your `lua/config/options.lua` that make changes to the current buffer?

**Reply by sshiradwade-synamedia** [reply]
my `lua/config/options.lua` is empty

**Reply by folke** [reply]
oh, it's in the mason UI. Something is trying to set an option when the mason ui is visible. I don't have this in my install, so maybe some plugin or a setting you did

**Reply by sshiradwade-synamedia** [reply]
Its fresh install, https://www.lazyvim.org/installation

1. I went to Mason UI by `:Mason`
2. Searched `tailwindcss-language-server` 
3. Hit Enter
4. Hit I

**Reply by folke** [reply]
`I` tries to go into insert mode at the start of the line.

You need to use lower `i` to install. Check the keymaps with `g?`

**Reply by sshiradwade-synamedia** [reply]
`i` works,  `g?` is helpful, did not noticed that one. Thanks Folke

**Reply by YvQue** [reply]
thanks folke

**Reply by BHofKS** [reply]
Still helping folks, @folke
